test/parallel/test-timers-unenroll-unref-interval.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const common = require('../common');
+const timers = require('timers');
+
+const keepOpen = setTimeout(() => {
+  common.fail('Test timed out.');
+}, common.platformTimeout(500)); // Keep event loop open.
```

Comment:
> Try this to see it fail

I think you've probably been going at this problem wrong then. If this fails (which it does... but only at more than 3x the load on my machine) then there is a very bad ordering bug in the timers impl when under load.